ducktape/cluster/remoteaccount.py

```diff
@@ -146,24 +146,19 @@ def operating_system(self):
 
     @property
     def logger(self):
-        if self._logger:
-            return self._logger
-        else:
-            return logging.getLogger(__name__)
+        if not self._logger:
+            self._logger = logging.getLogger(__name__)
+        return self._logger
 
     @logger.setter
     def logger(self, logger):
         self._logger = logger
 
-    def _log(self, level, msg, *args, **kwargs):
```

Comment:
doesn't removing this remove information from the log since we're no longer logging the account info that this is executing from? seems like that would reduce debuggability?

That might not be the ideal info to log (e.g. you probably want to know the current logical use of the node, e.g. connect worker 2, rather than the specific account, e.g. user@node1, but it's better than nothing?

Reply:
stacklevel seems to have been added in python 3.8, so we can't use that. we can use a logging adapter to add the output w/o modifying the stack though, so I did that.